src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,14 @@
-// your components will all go in this `component` directory.
-// feel free to change this component.js into TodoList.js
+import React from 'react';
+import Todo from "./Todo";
+
+const TodoList = props => {
+    return (
+        props.todos.map(todo => {
+            return (
+                <Todo todo={todo} key={todo.id} completed={todo.completed} completedHandler={props.completedHandler}/>
```

Comment:
you could pass all the todo data down at once in one object and unpackage them in the Todo component